sandbox/rocky/tf/algos/ddpg/replay_buffer.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import random
+
+
+class ReplayBuffer:
```

Comment:
The replay buffer from the ddpg in the theano tree is combined with ddpg algorithm in the same file. It should not include the ddpg file from the theano tree unless I separate them. Also, I think the sandbox tree should not use the functions in the theano tree since they use different platforms(theano and tensorflow).